src/Components/Path.php

```diff
@@ -72,10 +72,15 @@ public function getRelativePath(PathInterface $reference)
         }
         //deduce the number of similar segment according to the reference path
         $nb_common_segment = count($ref_path) - $index;
+        if ($nb_common_segment) {
+            $nb_segments = array_fill(0, $nb_common_segment, '..');
+        } else {
```

Comment:
When coding I try to follow as much as possible [object calisthenics](http://williamdurand.fr/2013/06/03/object-calisthenics/) . Could you remove the `else` keyword that way the code is much easier to understand